latest/index.bs

```diff
@@ -331,24 +317,1002 @@ Conforming readers:
+regions of interest, etc. SHOULD ensure that they are in the same coordinate system (same name, with identical axes) or can be
+transformed to the same coordinate system before doing analysis. See the example below.
+
+<div class=example>
+
+Two instruments simultaneously image the same sample from two different angles, and the 3D data from both instruments are
+calibrated to "micrometer" units. Two samples are collected ("sampleA" and "sampleB"). An analysis of sample A requires
+measurements from both instruments' images at certain points in space. Suppose a region of interest (ROI) is determined from the
+image obtained from instrument 2, but quantification from that region is needed for instrument 1. Since measurements were
+collected at different angles, a measurement by instrument 1 at the point with coordinates (x,y,z) may not correspond to the
+measurement at the same point in instrument 2 (i.e., it may not be the same physical location in the sample). To analyze both
+images together, they must be in the same coordinate system.
+
+The set of coordinate transformations ([[#trafo-md]]) encodes relationships between coordinate systems, specifically, how to
+convert points and images to different coordinate systems.  Implementations can apply the coordinate transform to images or
+points in coordinate system "sampleA_instrument2" to bring them into the "sampleA_instrument1" coordinate system. In this case,
+the ROI should be transformed to the "sampleA_image1" coordinate system, then used for quantification with the instrument 1
+image.
+
+```json
+"coordinateSystems" : [
+    {
+        "name" : "sampleA-instrument1",
+        "axes" : [
+            {"name": "x", "type": "space", "unit": "micrometer"},
+            {"name": "y", "type": "space", "unit": "micrometer"},
+            {"name": "z", "type": "space", "unit": "micrometer"}
+        ]
+    },
+    {
+        "name" : "sampleA-instrument2",
+        "axes" : [
+            {"name": "x", "type": "space", "unit": "micrometer"},
+            {"name": "y", "type": "space", "unit": "micrometer"},
+            {"name": "z", "type": "space", "unit": "micrometer"}
+        ]
+    }
+],
+"coordinateTransformations": [
+    {
+        "type": "affine":
+        "path": "../sampleA_instrument2-to-instrument1"
+        "input": "sampleA_instrument2",
+        "output": "sampleA_instrument1"
+    }
+]
+```
+
+</div>
+
+
+
+### "axes" metadata {#axes-md}
+
+"axes" describes the dimensions of a coordinate systems. It is a list of dictionaries, where each dictionary describes a dimension (axis) and:
+- MUST contain the field "name" that gives the name for this dimension. The values MUST be unique across all "name" fields.
+- SHOULD contain the field "type". It SHOULD be one of "array", "space", "time", "channel", "coordinate", or "displacement" but MAY take other values for custom axis types that are not part of this specification yet.
```

Comment:
Can we order them alphabetically? This would help structure it a bit better also w.r.t. the explanation below (see the relevant comments below).